instance/dig.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2021 The Dank Grinder authors.
+//
+// This source code has been released under the GNU Affero General Public
+// License v3.0. A copy of this license is available at
+// https://www.gnu.org/licenses/agpl-3.0.en.html
+//
+// Dig Functionality added by https://github.com/V4NSH4J
+
+package instance
+
+import (
+	"math/rand"
+	"regexp"
+
+	"github.com/dankgrinder/dankgrinder/discord"
+	"github.com/dankgrinder/dankgrinder/instance/scheduler"
+)
+
+func isSameLength(s1 string, s2 string) bool { // Function checks length of strings
+
+	if len(s1) == len(s2) {
+		return true
+	} else {
+		return false
+	}
+}
+
+func hasSameLetters(s1 string, s2 string) bool { // Function checks the letters of the strings (Scrambled and Unscrambled Words)
+	var counter int
+	for _, letter1 := range s1 {
+		for _, letter2 := range s2 {
+			if letter1 == letter2 {
+				counter++
+				break
+			}
+		}
+	}
+	if counter == len(s1) {
```

Comment:
I'll implement your suggestion but wouldn't cases like `hasSameLetters("foo", "fo")` not really matter because the word lengths are also being checked?

Reply:
If word lengths are checked that example would indeed be ruled out but other cases, such as `hasSameLetters("foo", "fof")` would still return `true` because of the way you implemented the checks.

Reply:
Oh Alright Gotcha!